src/lib/traitify.js

```diff
@@ -1,6 +1,14 @@
-import Traitify from "traitify-client";
-Traitify.ui = require("traitify-ui").default;
-Traitify.ui.client = Traitify;
-Traitify.__version__ = __VERSION__;
+
+/*global __VERSION__ */
+import TraitifyLib from "traitify-client";
 require("polyfills");
-export default Traitify;
\ No newline at end of file
+import ui from "traitify-ui";
+function Init(){
```

Comment:
There may be better ways to do it, I'd be down for a refactor after this, would love to just see this code out there though